test/reducers/players.js

```diff
@@ -68,6 +68,14 @@ describe('players reducer', () => {
     expect(playerState.playerId1.deck).to.have.lengthOf(24);
     expect(playerState.playerId1.hand).to.eql(['c25', 'c26', 'c27', 'c28', 'c29', 'c30']);
   });
+  it('should handle GAIN_MANA', () => {
+    const action1 = gainMana('playerId1', 1);
+    const action2 = gainMana('playerId2', 11);
+    let playerState = players(initialPlayerState, action1);
+    expect(playerState.playerId1.mana).to.eql(1);
+    playerState = players(initialPlayerState, action2);
```

Comment:
Change `initialPlayerState` to `playerState`

That way the actions build off each other.

I think for this you should have 3 actions to test:
* One that only takes in a player id to test the default mana param.
* One that takes in a mana value greater than 1
* And finally one that tests the hard limit at 10 max mana

I also like to add a test like this:
```
expect(playerState.playerId1).to.contain.all.keys(['id', 'name']);
```
This makes sure the state has a id and name key. I put this check in to make sure I'm not wiping all the other properties from the player object :P